collection-scripts/gather_services_status

```diff
@@ -87,6 +92,29 @@ get_heat_status() {
     ${BASH_ALIASES[os]} orchestration service list > "$HEAT_PATH"/service_list
 }
 
+# Nova service gathering - sevices, hypervisors, cells, host mappings, allocation audit
+get_nova_status() {
+    local NOVA_PATH="$BASE_COLLECTION_PATH/ctlplane/nova"
+    mkdir -p "$NOVA_PATH"
+    run_bg ${BASH_ALIASES[os]} compute service list '>' "$NOVA_PATH"/service_list
+    run_bg ${BASH_ALIASES[os]} hypervisor list '>' "$NOVA_PATH"/hypervisor_list
+    run_bg /usr/bin/oc -n openstack exec -t nova-cell0-conductor-0 -- nova-manage cell_v2 list_cells '>' "$NOVA_PATH"/cell_list
+    run_bg /usr/bin/oc -n openstack exec -t nova-cell0-conductor-0 -- nova-manage cell_v2 list_hosts '>' "$NOVA_PATH"/host_list
+    run_bg ${BASH_ALIASES[os]} aggregate list --long '>' "$NOVA_PATH"/aggregate_list
+}
```

Comment:
i still think it would be nice to include a flavour detail list output
other then that i think this is a good start so I'm ok with differing that to a spereate pr.